authui/src/authflowv2/dismissKeyboard.ts

```diff
@@ -0,0 +1,27 @@
+import { Controller } from "@hotwired/stimulus";
+
+/**
+ * On iOS, the keyboard is treated as overlay rather than part of the viewport.
+ * This causes an extra padding to be added to the bottom of the viewport when the keyboard is open.
+ *
+ * This controller listens for scroll events and dismisses the keyboard when user scrolls on
+ * the body or non-`overflow:auto/scroll` element.
+ */
+export class DismissKeyboardOnScrollController extends Controller {
+  connect() {
+    window.addEventListener("scroll", this.dismissKeyboard);
```

Comment:
Use `capture` to listen to all possible scroll events?

Reply:
Currently set `capture` to false to avoid closing keyboard when scrolling in the options container. 

Otherwise the keyboard will be dismissed when user is typing in searchbar since there is autoscrolling

Reply:
I see